src/lib/seam/connect/routes/workspaces.ts

```diff
@@ -0,0 +1,36 @@
+// TODO: This file is generated from route spec
+import type { Routes } from '@seamapi/types/connect'
+import type { Axios } from 'axios'
+import type { SetNonNullable } from 'type-fest'
+
+export class Workspaces {
+  client: Axios
+
+  constructor(client: Axios) {
+    this.client = client
+  }
+
+  async get(params: WorkspacesGetParams = {}): Promise<Workspace> {
+    const {
+      data: { workspace },
+    } = await this.client.get<WorkspacesGetResponse>('/workspaces/get', {
+      params,
+    })
+    return workspace
+  }
+}
+
+export type WorkspacesGetParams = SetNonNullable<
+  Required<Routes['/workspaces/get']['commonParams']>
+>
+
+export type WorkspacesGetResponse = SetNonNullable<
+  Required<Routes['/workspaces/get']['jsonResponse']>
+>
+
+// UPSTREAM: Should come from @seamapi/types/connect
+// import type { Workspace } from @seamapi/types
```

Comment:
Need to ensure all types are exported from public type, so likely this will be added later